services/event-application-service/src/commands/domain.h

```diff
@@ -0,0 +1,36 @@
+#ifndef COMMANDS_DOMAIN_H
+#define COMMANDS_DOMAIN_H
+
+#include <string>
+
+namespace eas::commands {
+struct CreateForm {
+  std::string data;
+};
+
+struct UpdateForm {
+  std::string data;
+};
+
+struct DeleteFormAndResponses {
+  std::string data;
+};
+
+struct CreateResponse {
+  std::string data;
+};
+
+struct UpdateRespone {
+  std::string data;
+};
```

Comment:
Still, one base class to store data

Reply:
I don't think so. It's just stubs for now. I will add required field with next pro

Reply:
Didn't know it, ok